virtcontainers/device/manager/utils_test.go

```diff
@@ -89,3 +91,44 @@ func TestIsVhostUserSCSI(t *testing.T) {
 		assert.Equal(t, d.expected, isVhostUserSCSI)
 	}
 }
+
+func TestIsLargeBarSpace(t *testing.T) {
+	assert := assert.New(t)
+
+	// File not exist
+	bs, err := isLargeBarSpace("/abc/xyz/123/rgb")
+	assert.Error(err)
+	assert.False(bs)
+
+	f, err := ioutil.TempFile("", "pci")
+	assert.NoError(err)
+	defer f.Close()
+	defer os.RemoveAll(f.Name())
+
+	type testData struct {
+		resourceInfo string
+		error        bool
+		result       bool
+	}
+
+	for _, d := range []testData{
+		{"", false, false},
+		{"\t\n\t  ", false, false},
+		{"abc zyx", false, false},
+		{"abc zyx rgb", false, false},
+		{"abc\t       zyx     \trgb", false, false},
+		{"0x00015\n0x0013", false, false},
+		{"0x00000000c6000000 0x00000000c6ffffff 0x0000000000040200", false, false},
+		{"0x0000383800000000 0x0000383bffffffff 0x000000000014220c", false, true},
```

Comment:
If we only care about the first two fields, could you add another entry here for a successful call with only two fields?